dmoj/executors/R.py

```diff
@@ -16,9 +16,9 @@ class Executor(ScriptDirectoryMixin, ScriptExecutor):
 
     if os.name != 'nt':
         syscalls = ['mkdir', 'setup', 'fork', 'waitpid', 'wait4', 'getpgrp', 'execve',
-                    ('statfs64', ACCESS_DENIED), ('statfs', ACCESS_DENIED)]
+                    ('statfs64', ACCESS_DENIED), ('statfs', ACCESS_DENIED), 'newfstatat', 'unlinkat', 'openat']
```

Comment:
This is definitely unsafe, it'd let a judged process open and unlink any file the user it is running under has access to (this would, for example, include problem data or the judge code itself).

R is not really feasible to run reliably in a `ptrace` sandbox, unfortunately. It updates frequently, always breaking in new creative ways. On the other hand, it hasn't been used so solve more than "Hello, World!" on dmoj.ca in 2 years, so it likely doesn't matter.